webapp/components/LoginForm/LoginForm.vue

```diff
@@ -20,18 +24,20 @@
           name="email"
           icon="envelope"
         />
-        <ds-input
-          v-model="form.password"
-          :disabled="pending"
-          :placeholder="$t('login.password')"
-          icon="lock"
-          icon-right="question-circle"
-          name="password"
-          type="password"
-        />
+          <ds-input
+            v-model="form.password"
+            :disabled="pending"
+            :placeholder="$t('login.password')"
+            icon="lock"
+            icon-right="question-circle"
+            name="password"
+            type="password"
+            ref="passwordInput"
+          />
         <nuxt-link to="/password-reset/request">
           {{ $t('login.forgotPassword') }}
         </nuxt-link>
+        <text class="caps-warning" v-show="caps" >CAPS LOCK ENABLED!</text>
```

Comment:
Please localize the text. German would be: `Feststelltaste ist eingerastet`

Reply:
@Mogge Is there an abbreviation in german for the capslock? Or should we use a symbol instead? The localization is causing issues with the styling and its consistency between the locales.

Reply:
`Capslock ist an` could be an alternative